src/components/ScriptExplorer/SignatureImporter.jsx

```diff
@@ -576,8 +596,20 @@ function mapStateToProps(state, ownProps) {
       signatureImporter: state.spend.signatureImporters[ownProps.number],
       fee: state.spend.transaction.fee,
       txid: state.spend.transaction.txid,
+      walletName: state.wallet.common.walletName,
     },
     ...state.spend.transaction,
+    requiredSigners: state.settings.requiredSigners,
+    extendedPublicKeys: Object.values(
+      state.quorum.extendedPublicKeyImporters
+    ).map((key) => ({
+      bip32Path:
+        key.bip32Path.toLowerCase() === "unknown"
+          ? "m/45'/0/0/0"
```

Comment:
I actually added a utility for getting the masked paths so I might just use this here. I might have just added this hard coded to get unblocked and basically since it's unknown, and therefore masked, the real path _shouldn't_ matter, but yeah, proper masking is probably best.

Reply:
Good call. Cleaned up in [75e1668](https://github.com/unchained-capital/caravan/pull/288/commits/75e1668df09a6beb1cec80cfc5f099647bccd279)